credits.html

```diff
@@ -0,0 +1,40 @@
+<html>
+<head>
+	<title>
```

Comment:
@KFernando Titles are usually put on a single line since they are so short like this:
```<title>Some Title Here</title>```